modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -1879,7 +1879,7 @@ def filter(self, axis: Union[Axis, int], condition: Callable) -> "PandasDatafram
             new_partitions,
             *new_axes,
             *new_lengths,
-            self.dtypes if axis == 0 else None,
+            self.dtypes if axis.value == 1 else None,
```

Comment:
```suggestion
            self.dtypes if axis == Axis.ROW_WISE else None,
```

Reply:
In that case it's needed to use `Axis.COL_WISE`.

Reply:
right